common/search/src/main/scala/weco/api/search/config/builders/PipelineElasticClientBuilder.scala

```diff
@@ -18,14 +19,27 @@ object PipelineElasticClientBuilder {
 
   def apply(
     serviceName: String,
-    pipelineDate: String = PipelineClusterElasticConfig.pipelineDate
+    pipelineDate: String = PipelineClusterElasticConfig.pipelineDate,
+    isDev: Boolean = false
```

Comment:
Perhaps worth using an `environment` string to pattern match on instead? Generally I am opposed to boolean flags...

Reply:
Agreed.